src/pep8ext_naming.py

```diff
@@ -308,10 +308,9 @@ def arg_name(arg):
                 return node, arg
 
         for arg, name in arg_name(node.args.vararg), arg_name(node.args.kwarg):
-            if name is not None:
-                if not name.islower():
-                    yield self.err(arg, 'N803', name=name)
-                    return
+            if name is not None and not name.islower() and name != '_':
```

Comment:
Is `name.islower()` any better than `name.lower() == name`? I don't know that we should just exclude `_` wholesale. For reference,

```
>>> '_'.lower()
'_'
>>> '_'.lower() == '_'
True
>>> '_'.islower()
False
```

Reply:
Yes, that's a good suggestion. I just pushed a commit which changes all of the relevant checks to use that approach.